### ISSUE TITLE
Plugin not listed on Musescore Website

### ISSUE THREAD

**Issue by SamuelMiller** [OMerkel/tin_whistle#2]
Hello,
I wasn't able to find your lovely plugin at https://musescore.org/en/plugins
I happened to just stumble upon the github version through Google.
I suggest you submit it to musescore.

Thanks, Sam

**Comment by tthfry** [comment]
Yeah, I just stumbled over this *after* having updated the other plugin to MuseScore 2.2...

There are 1 or 2 things I like better about either plugin. This one obviously has a nicer license. ;-) Plus, the font is licensed by the same guy and included.
